ecosystem/resources/members/quantumcat.toml

```diff
@@ -48,117 +48,117 @@ passed = false
 test_type = "development"
 passed = false
 package = "qiskit-terra"
-package_version = "0.21.0"
-terra_version = "0.21.0"
+package_version = "0.37.0"
+qiskit_version = "0.37.0"
 timestamp = 1678827878.289239
 
 [[historical_test_results]]
 test_type = "stable"
 passed = false
 package = "qiskit-terra"
-package_version = "0.17.4"
-terra_version = "0.17.4"
+package_version = "0.27.0"
+qiskit_version = "0.27.0"
 timestamp = 1678827878.289244
 logs_link = "https://github.com/qiskit-community/ecosystem/actions/runs/2998938614"
 
 [[historical_test_results]]
 test_type = "development"
 passed = false
 package = "qiskit-terra"
-package_version = "0.22.0"
-terra_version = "0.22.0"
+package_version = "0.39.0"
+qiskit_version = "0.39.0"
 timestamp = 1678827878.289247
-logs_link = "https://github.com/qiskit-community/ecosystem/actions/runs/3180920000"
+logs_link = "https://github.com/qiskit-community/ecosystem/actions/runs/3180.36.00"
```

Comment:
Do you know why this changed? The URL is now broken. I'm curious the cause because it may have impacted other files.

Reply:
Nice see !

It's seems my sed didn't care if it has a `.` or not and so just replace any string that can look like the version dot or not.